build.gradle

```diff
@@ -340,120 +339,94 @@ repositories {
+  implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.47'
   implementation group: 'org.flywaydb', name: 'flyway-core'
   implementation group: 'org.flywaydb', name: 'flyway-database-postgresql'
   implementation group: 'org.postgresql', name: 'postgresql'
 
-  implementation group: 'com.google.guava', name: 'guava', version: '32.1.3-jre'
-  //Added org.glassfish to support javax.el
-  implementation group: 'org.glassfish', name: 'javax.el', version: '3.0.0'
-  implementation group: 'javax.el', name: 'javax.el-api', version: '3.0.0'
+  implementation group: 'com.google.guava', name: 'guava', version: '33.3.1-jre'
+  implementation group: 'jakarta.el', name: 'jakarta.el-api', version: '6.0.1'
   implementation group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '6.2.1'
 
-  implementation group: 'com.microsoft.azure', name: 'applicationinsights-runtime-attach', version: '3.6.0'
-  implementation group: 'com.microsoft.azure', name: 'applicationinsights-core', version: '3.6.0'
+  implementation group: 'com.microsoft.azure', name: 'applicationinsights-core', version: '3.6.2'
 
   implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: versions.log4j
   implementation group: 'org.apache.logging.log4j', name: 'log4j', version: versions.log4j
   implementation group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: versions.log4j
 
-  implementation group: 'com.azure', name: 'azure-core', version: '1.53.0'
-  implementation group: 'com.azure', name: 'azure-messaging-servicebus', version: '7.17.4'
+  implementation group: 'com.azure', name: 'azure-core', version: '1.54.1'
+  implementation group: 'com.azure', name: 'azure-messaging-servicebus', version: '7.17.7'
   implementation group: 'com.google.code.gson', name: 'gson', version: '2.11.0'
 
   implementation group: 'io.github.openfeign.form', name: 'feign-form', version: '3.8.0'
   implementation group: 'io.github.openfeign.form', name: 'feign-form-spring', version: '3.8.0'
-  implementation "io.github.openfeign:feign-httpclient:11.10"
+  implementation "io.github.openfeign:feign-httpclient:13.5"
   implementation "com.github.hmcts.java-logging:logging:${versions.reformLogging}"
-
   implementation group: 'org.springdoc', name: 'springdoc-openapi-starter-webmvc-ui', version: '2.6.0'
-  implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.8.0'
-  implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-bootstrap'
-  // https://mvnrepository.com/artifact/net.minidev/json-smart
-  implementation group: 'net.minidev', name: 'json-smart', version: '2.5.1'
 
+  testImplementation group: 'com.github.hmcts', name: 'rd-commons-lib', version: '0.1.0'
 
-  testImplementation ('com.github.hmcts:rd-commons-lib:0.0.18'){
-    exclude group: 'org.springframework.boot', module: 'spring-boot-starter-web'
-  }
-  //Fix for CVE-2021-29425
-  implementation 'commons-io:commons-io:2.17.0'
```

Comment:
a run of:
`./gradlew dependencyInsight --no-single-path --rerun --configuration runtimeClasspath --dependency commons-io`
reveals that version 2.11 is being used. please ensure at least version 2.17.0 is used due to CVE-2021-29425
(possibly include resolution strategy to address transitive dependencies)

Reply:
Thanks for looking into these. I checked with owasp dependency checker and that didn't report any issues.. Updated to use non-vulnerable versions.